tests/crashes/124583.rs

```diff
@@ -1,5 +0,0 @@
-//@ known-bug: rust-lang/rust#124583
-
-fn main() {
-    let _ = -(-0.0f16);
-}
```

Comment:
Could you add   Fixes #124583 to pr header so that this is auto closed? thx!

Reply:
I had it in a commit message and thought it would snag it there, but added just in case :)

Reply:
ah I didn't check the commits :upside_down_face: should be good enough then :)